io/plugins/eu.esdihumboldt.hale.io.validation/plugin.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.action">
+      <action
+            categoryName="Validator configurations"
+            id="eu.esdihumboldt.hale.io.validation.read"
+            name="Load validator configuration"
+            resourceName="validator configuration"
+            type="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl">
```

Comment:
This should be the interface - you state here what kind of I/O providers are allowed for the action.